java/cuvs-java/src/main/java/com/nvidia/cuvs/panama/cuvsCagraSearchParams.java

```diff
@@ -0,0 +1,697 @@
+/*
```

Comment:
I know we've discussed having this automatically generate on each build in a future iteraiton. Can you please create a cuVS Github issue to do that?